src/content/terraform/product-landing.json

```diff
@@ -52,6 +52,27 @@
 		]
 	},
 	"blocks": [
+		{
+			"type": "heading",
+			"heading": "Best practices",
```

Comment:
I 100% defer to your expertise on this one, but I see the "Featured Documentation" header, "Configuration Language" card, etc seem to use title case instead of sentence case, should we match the existing style?